internal/jennies/java/rawtypes.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/grafana/cog/internal/tools"
 )
 
+const projectPath = "src/main/java/com/grafana/foundation"
+const packagePath = "com.grafana.foundation"
```

Comment:
Shouldn't these come from the jenny's configuration?

Reply:
To be able to change them for any reason?

Reply:
In case cog - which is mostly not grafana-specific - is used in another context, yeah :)

Reply:
I just moved everything into the configuration.